docs/examples/getDataMoverResources.go

```diff
@@ -0,0 +1,407 @@
+		Group:    "snapshot.storage.k8s.io",
+		Version:  "v1",
+		Resource: "volumesnapshots",
+	}
+	VolumeSnapshotContentResource = schema.GroupVersionResource{
+		Group:    "snapshot.storage.k8s.io",
+		Version:  "v1",
+		Resource: "volumesnapshotcontents",
+	}
+	// Old VSM/Volsync DataMover
+	VolumeSnapshotBackupResource = schema.GroupVersionResource{
+		Group:    "datamover.oadp.openshift.io",
+		Version:  "v1alpha1",
+		Resource: "volumesnapshotbackups",
+	}
+	VolumeSnapshotRestoreResource = schema.GroupVersionResource{
+		Group:    "datamover.oadp.openshift.io",
+		Version:  "v1alpha1",
+		Resource: "volumesnapshotrestores",
+	}
+	ReplicationSourceResource = schema.GroupVersionResource{
+		Group:    "volsync.backube",
+		Version:  "v1alpha1",
+		Resource: "replicationsources",
+	}
+	ReplicationDestinationResource = schema.GroupVersionResource{
+		Group:    "volsync.backube",
+		Version:  "v1alpha1",
+		Resource: "replicationdestinations",
+	}
+	// New Velero Built-in DataMover
+	dataUploadResource = schema.GroupVersionResource{
+		Group:    "velero.io",
+		Version:  "v2alpha1",
+		Resource: "datauploads",
+	}
+	dataDownloadResource = schema.GroupVersionResource{
+		Group:    "velero.io",
+		Version:  "v2alpha1",
+		Resource: "datadownloads",
+	}
+)
+
+var cli = &cobra.Command{
+	Use: "go run docs/examples/getDataMoverResources.go",
+	Long: `Check the OADP DataMovers Custom Resources in real time.
+
+Select either Velero Built-in DataMover or VSM/Volsync DataMover Resources.
+
+You need to log in to your cluster and deploy OADP with DataMover prior to run this script`,
+	Args: cobra.NoArgs,
+	Example: `  # Check the Velero Built-in DataMover Backup Resources
+  go run docs/examples/getDataMoverResources.go -b
+
+  # Check the VSM/Volsync DataMover Backup Resources
+  go run docs/examples/getDataMoverResources.go -b -v=false
+
+  # Check the Velero Built-in DataMover Resources with details
+  go run docs/examples/getDataMoverResources.go -r -d`,
+	Run: func(cmd *cobra.Command, args []string) {
+		getClients()
+		if builtInDataMover {
+			if backup {
+				newBackupSummary()
+			}
+			if restore {
+				newRestoreSummary()
+			}
+		} else {
+			if backup {
+				oldBackupSummary()
+			}
+			if restore {
+				oldRestoreSummary()
+			}
+		}
+	},
+}
+
+func init() {
+	cli.Flags().BoolVarP(&backup, "backup", "b", false, "Check a backup")
+	cli.Flags().BoolVarP(&restore, "restore", "r", false, "Check a restore")
+	cli.MarkFlagsMutuallyExclusive("backup", "restore")
+	// TODO https://github.com/spf13/cobra/pull/1952
+	// cli.MarkFlagsOneRequired("backup", "restore")
+	cli.Flags().BoolVarP(&details, "details", "d", false, "Print a list of the relevant backup/restore CR's")
+	cli.Flags().StringVarP(&namespace, "namespace", "n", "openshift-adp", "Namespace OADP was deployed")
+	cli.Flags().BoolVarP(&builtInDataMover, "velero", "v", true, "Check for Velero Built-in DataMover Resources, if true, otherwise, check for VSM/Volsync DataMover Resources")
```

Comment:
Added flag to select between Velero Built-in DataMover and VSM/Volsync DataMover :smile: